docs/recipes/vue.md

```diff
@@ -40,9 +45,11 @@ Vue.config.productionTip = false;
 // Setup vue files to be processed by `require-extension-hooks-vue`
 hooks('vue').plugin('vue').push();
 // Setup vue and js files to be processed by `require-extension-hooks-babel`
-hooks(['vue', 'js']).plugin('babel').push();
+hooks(['vue', 'js']).exclude(({filename}) => filename.includes('node_modules')).plugin('babel').push();
```

Comment:
This is a fragile way to test for a `node_modules` directory. It will also match paths like `foo-node_modules-bar`.